integration-tests/chopsticks/src/network/hydraDx.ts

```diff
@@ -37,12 +44,13 @@ export function assignKiltTokensToAccounts(addr: string[], balance: bigint = ini
 	}
 }
 
+// Sibling Sovereign Account
+export const siblingSovereignAccount = '5Eg2fntQqFi3EvFWAf71G66Ecjjah26bmFzoANAeHFgj9Lia'
```

Comment:
I think it would also be better to group "required" info for each parachain, such as its sibling account and para ID into its own object with its own defined type. Everything else is extra, which can extend the basic object, kinda like you did for the tests setup.